### PR TITLE
Add lm-eval for tpu7x.

### PR DESCRIPTION
**Pull request by QiliangCui**

### CODE REVIEW

File: scripts/scheduler/hourly_run.sh [hunk 1]
```diff
@@ -110,6 +110,10 @@ if [[ "$HOUR_NOW" == "01" || "$HOUR_NOW" == "13" ]]; then
   echo "./scripts/scheduler/create_job.sh ./cases/accuracy_jax.csv \"\" $TAG JAX_ACCURACY TPU_INFERENCE \"TPU_BACKEND_TYPE=jax;\""
   ./scripts/scheduler/create_job.sh ./cases/accuracy_jax.csv "" $TAG JAX_ACCURACY TPU_INFERENCE "TPU_BACKEND_TYPE=jax;"
 
+  # TT v7 accuracy (mmlu_tpu on tpu7x)
```
Comment (QiliangCui) [inline scripts/scheduler/hourly_run.sh]: don't enable in hourly_run. it is still debugging.

File: cases/accuracy_tt_v7.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Device,Model,MaxNumSeqs,MaxNumBatchedTokens,TensorParallelSize,MaxModelLen,Dataset,InputLen,OutputLen,ExpectedETEL,NumPrompts
+tpu7x-8,Qwen/Qwen3-Coder-480B-A35B-Instruct-FP8,512,4096,8,8192,mmlu_tpu,0,0,,,
```
Comment (QiliangCui) [inline cases/accuracy_tt_v7.csv]: rather than add a new mmlu_tpu, does it make more sense to just use mmlu where everywhere is already setup?
